examples/main.tf

```diff
@@ -0,0 +1,47 @@
+data "aws_iam_policy_document" "read-a-bucket" {
+  statement {
+    sid    = "AllowReadOfABucket"
+    effect = "Allow"
+
+    actions = [
+      "s3:GetObject",
+    ]
+
+    resources = [
+      "arn:aws:s3:::public-bucket/*",
```

Comment:
Should avoid using wildcard * in the example, since it will promote user to use the wildcard

Reply:
The wildcard is for the content of the bucket though which should be fine. In most cases, it's safer to use * to read contents of the bucket